yourgov/components/FormMobileFoodVendorPermit/steps/FormStep10Review.tsx

```diff
@@ -277,14 +278,30 @@ export function FormStep10Review() {
 
 			{/** Upload documents */}
 			<Stack gap={1.5}>
-				<H2>TODO - {formSteps[8].label}</H2>
+				<H2>{formSteps[8].label}</H2>
 				{validation.step9.success ? (
 					<FormDefinitionList>
 						<FormDefinitionListItem
 							label="Documents"
-							value={formState.steps?.step9?.cuisine
-								?.map((item) => item?.label)
-								.join(', ')}
+							value={
+								<ul
```

Comment:
Still think this should be a table

Reply:
Ah yeah, I forgot about this one.

- [ ] tabulate review step to match Employees